appserver/tests/cts_smoke/run_test.sh

```diff
@@ -198,8 +203,11 @@ run_test_id(){
 	if [[ $1 = "cts_smoke_all" ]]; then
 		test_run_cts_smoke
 		result=$WORKSPACE/results/smoke.log
-	elif [[ $1 = "servlet_tck_all" ]]; then
-		test_run_servlet_tck
+	elif [[ $1 = "servlet_tck_"* ]]; then
```

Comment:
This is already tested against 'all' target and it passed. I am not sure about any other use case where it should fail.